Typos -  Developer Toolkit section (#10404)
circleci/circleci-docs: docs/guides/modules/toolkit/pages/api-developers-guide.adoc

 -d '{ "branch": "my-branch" }'
----

> . Let's move on to a more complex example: triggering a pipeline and passing a parameter that can be dynamically substituted into your configuration. In this example, you will pass a Docker image tag to the `docker` executor key.

Reviewer: ```suggestion
. The next example is more complex: Triggering a pipeline and passing a parameter that can be dynamically substituted into your configuration. In this example, you will pass a Docker image tag to the `docker` executor key.
```
